gdal/swig/csharp/GNUmakefile

```diff
@@ -89,9 +89,11 @@ samples:
 test:
 	LC_ALL=C mono createdata.exe Data pointlayer
 	LC_ALL=C mono ogrinfo.exe Data/pointlayer.shp
-	LC_ALL=C mono OSRTransform.exe
 	LC_ALL=C mono GDALWrite.exe Data/sample.tif
 	LC_ALL=C mono GDALDatasetWrite.exe Data/sample1.tif
 	LC_ALL=C mono GDALCreateCopy.exe Data/sample.tif Data/sample2.tif
 	LC_ALL=C mono GDALOverviews.exe Data/sample.tif NEAREST 2 4
 	LC_ALL=C mono GDALInfo.exe Data/sample.tif
+
+alltest: test
```

Comment:
could we instead rename the ```test``` target as being ```vagrant_safe_test``` and the new ```alltest``` as ```test```, and Vagrant would run ```vagrant_safe_test``` ? That way, in normal context, make test would still run everything